superset/models.py

```diff
@@ -1278,6 +1278,11 @@ def query(  # sqla
         if granularity not in self.dttm_cols:
             granularity = self.main_dttm_col
 
+        # Database spec supports join-free timeslot grouping
+        time_groupby_inline = False
```

Comment:
this whole section is not necessary, as we can use `self.database.db_engine_spec.time_groupby_inline` in place of the local var